editor/modes/visual-editor/block.js

```diff
@@ -133,6 +134,17 @@ class VisualEditorBlock extends wp.element.Component {
 
 		const { onSelect, onStartTyping, onHover, onMouseLeave, onFocus, onInsertAfter } = this.props;
 
+		// Normalize controls as an array of arrays (toolbars of controls)
```

Comment:
Maybe something like `<div className="editor-toolbar__group">` should move to a separate component?